projects/m5/001-budget-app/js/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "budget_app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest"
```

Comment:
You can improve the result by adding the `--coverage` option ;-)
`"test": "jest --coverage`